test/integration/options/watch.spec.js

```diff
@@ -343,5 +345,31 @@ describe('--watch', function() {
       it('mochaHooks.afterAll runs as expected', setupHookTest('afterAll'));
       it('mochaHooks.afterEach runs as expected', setupHookTest('afterEach'));
     });
+
+    it('should not leak event listeners', function() {
+      this.timeout(20000);
+      const testFile = path.join(tempDir, 'test.js');
+      copyFixture(DEFAULT_FIXTURE, testFile);
+
+      return expect(
+        runMochaWatchAsync(
+          [testFile],
+          {cwd: tempDir, stdio: 'pipe'},
+          async () => {
+            const iterations = new Array(15);
```

Comment:
Might be faster and more robust to set this to `emitter.getMaxListeners() + 1`. Maybe it works by setting the max listener count to a low number e.g. 1 or 2 which should reduce test runtime.

Reply:
would rather just use the default settings, so yeah, `emitter.getMaxListeners()` sounds better